products/batch_exports/backend/temporal/batch_exports.py

```diff
@@ -426,7 +429,14 @@ async def finish_batch_export_run(inputs: FinishBatchExportRunInputs) -> None:
     logger = LOGGER.bind()
     external_logger = EXTERNAL_LOGGER.bind()
 
-    not_model_params = ("id", "team_id", "batch_export_id", "failure_threshold", "failure_check_window")
+    not_model_params = (
+        "id",
+        "team_id",
+        "batch_export_id",
+        "failure_threshold",
+        "failure_check_window",
+        "bytes_exported",  # TODO: once we add this to the model we can remove this
```

Comment:
This doesn't seem to be used anywhere, so, why not just omit it until the model is updated?

Reply:
well I either needed to do this, or not add `bytes_exported` to the `FinishBatchExportRunInputs` yet. I don't really mind too much either way - we either need to do more work in this PR or in the follow up PR